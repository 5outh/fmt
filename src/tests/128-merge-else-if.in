<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,MergeElseIf

if($a){
	$b;
}else if($b){
	$c;
}
