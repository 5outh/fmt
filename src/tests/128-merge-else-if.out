<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,MergeElseIf

if ($a) {
	$b;
} elseif ($b) {
	$c;
}
